containers/unit-tests/run.sh

```diff
@@ -35,47 +40,10 @@ git clone /source /tmp/source
 [ ! -d /source/node_modules ] || cp -r /source/node_modules /tmp/source/
 cd /tmp/source
 
-./autogen.sh --prefix=/usr --enable-strict --with-systemdunitdir=/tmp
-
-make all
+containers/unit-tests/build.sh
```

Comment:
That apparently happens to work for the workflow invocation, but it would definitively break the `/source/containers/unit-tests/run.sh` invocation in interactive containers, as per containers//unit-tests/README.md. Can you please add a `MYDIR=$(dirname $0)` at the top and call build.sh from that instead?

Reply:
This should be fine.  The line directly before is `cd /tmp/source`, so we are always in the same location, at the top of our cloned copy of the source tree.